Add dora-lerobot node into dora

Ignavar opened the pull request:
migrated node-hub from dora-lerobot to dora and fixed formatting of the toml files as well

haixuanTao commented:
FYI, in case of doubt, I believe chatgpt or deepseek should be able to convert the original pyproject.toml to the desired pyproject.toml given an example.

haixuanTao commented:
Yes if you don't mind!

You can use the fix flag sometimes to fix it automatically.

Ignavar commented:
Hey,

I fixed the warnings, but I ran into an issue where the new modules didn't have test folders, causing the tests to fail since the test scripts couldn't be found. I checked the existing modules and added test files based on those examples, which primarily involved calling the main.py file of each module.

Now, I'm encountering a runtime error related to missing argument values. Could you guide me on how to proceed?

Thanks!

haixuanTao commented:
Yes so, I would believe that some of the main function will requires some hardware being connected. What you can do for now is just to add a dummy test like https://github.com/dora-rs/dora/blob/1b95bf478a8bbdcfe9392d442eb0b3ee1b90bf1b/node-hub/dora-reachy2/tests/test_dora_reachy2.py 

to skip pytest for now.

You can also add some default value if you think that it can pass pytest.

FYI, you can run test locally with:

```bash
cd node-hub/dora-pyaudio # Change for the one you want to use
../../.github/workflows/node_hub_test.sh
```

Ignavar commented:
Hey, I'm running into an error when trying to run the tests locally. I've checked my virtual environment, and the required package is definitely installed. Could you guide me on how to resolve this? It would really help with future commits. Apologies for bothering you again!

![image](https://github.com/user-attachments/assets/2be03c8d-ebde-4625-91fd-3fb94c557037)
![image](https://github.com/user-attachments/assets/07006532-ac6d-4324-9e32-44ed5b921ace)

haixuanTao commented:
No bother at all!

Could you make sure that you are running outside of a virtual environment.

You can try to deactivate them with:

```
conda deactivate
# or
deactivate
```

Ignavar commented:
I tried running it outside the virtual environment and still encountered the exact same error. Also, could you kindly review this pull request?

haixuanTao reviewed (approved):
Looks great! Thanks!
